ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -23,8 +23,9 @@ export interface AccountValue {
   createdAt: HeadValue | null
 }
 
```

Comment:
We should probably consider splitting this type...

Reply:
Agreed; we'll be getting to the point where only ~half the fields from `AccountValue` will be the same in this type